ui/app/templates/vault/cluster/clients/index.hbs

```diff
@@ -0,0 +1,47 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      Vault Client Count
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<div class="tabs-container box is-bottomless is-marginless is-fullwidth is-paddingless" data-test-pricing-metrics>
+  <nav class="tabs">
+    <ul>
+      <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='current' start=null end=null}} @tagName="li" @activeClass="is-active">
+        <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='current' start=null end=null}} data-test-usage-tab={{true}}>
+          Current month
+        </LinkTo>
+      </LinkTo>
+      <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='history'}} @tagName="li" @activeClass="is-active">
+        <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='history'}} data-test-usage-tab={{true}}>
+          Monthly history
+        </LinkTo>
+      </LinkTo>
+      {{#if model.config.configPath.canRead}}
+        <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='config' start=null end=null}} @tagName="li" @activeClass="is-active">
+          <LinkTo @route="vault.cluster.clients.index" @query={{hash tab='config' start=null end=null}} data-test-configuration-tab={{true}}>
+            Configuration
+          </LinkTo>
+        </LinkTo>
+      {{/if}}
+    </ul>
+  </nav>
+</div>
+
```

Comment:
Above you have one space between the end of PageHeader and the next chunk of code, maybe keep consistent here and do one space between these two chunks as well.